### PR TITLE
center hw pictures & add mkdocs-requirements.txt

### PR DESCRIPTION
**Pull request by docgalaxyblock**
Happy 🐇🥚

### CODE REVIEW

**Comment by docgalaxyblock** [comment]
Just want to mention that https://github.com/LMS-Community/lms-server-repository/blob/master/index.md needs a rebranding too ^^

**Comment by terual** [comment]
Thanks!!

File: docs/players-and-controllers/SLIMP3.md [hunk 1]
```diff
@@ -3,12 +3,16 @@ layout: default
 title: SLIMP3
 ---
 
+# SLIMP3
+
+<figure markdown="span">
```
Comment (michaelherger) [inline docs/players-and-controllers/SLIMP3.md:8]: May I ask what that instruction is for (besides the fact that it would center an image)?

**Comment by docgalaxyblock** [comment]
@michaelherger https://squidfunk.github.io/mkdocs-material/reference/images/#image-alignment

**Pull request merged**